bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -89,10 +85,23 @@
 {% block content %}
 <main role="main">
   <header id="main-feature">
-      <h1 class="large">{{_('Check Your Plugins')}}</h1>
+      <a href="{{ url('firefox') }}">{{ high_res_img('plugincheck/firefox-logo.png', {'alt': 'Mozilla Firefox', 'width': '145', 'class': 'firefox-logo'}) }}</a>
+      <h2 class="large">{{_('Check Your Plugins')}}</h2>
       <p class="preamble">{{_('Keeping your plugins up to date helps Firefox run safely and smoothly.')}}</p>
   </header>
 
+  <section class="newsletter-container">
+    {% if LANG == 'en-US' %}
+      <header>
+        <h3>{{ _('<span class="firefox">Firefox</span> + You') }}</h3>
+        <h4>{{ _('Get Firefox tips, tricks, news and more') }}</h4>
+      </header>
+      {{ email_newsletter_form(button_class='white') }}
+    {% else %}
```

Comment:
Also here @flodolo, do we still need the conditionals or do we have enough coverage to remove them? Thx